components/candle/challengeService.ts

```diff
@@ -418,6 +418,10 @@ export abstract class ChallengeRegistry {
             return gameGroups.get("GLOBAL_ESCALATION_CHALLENGES")?.get(groupId)
         }
 
+        if (groupId?.includes("peacock")) {
```

Comment:
Worried this might conflict with Paris

Reply:
Challenge groups don't have any location-specific titles in there.